samigo/samigo-app/src/webapp/jsf/delivery/item/deliverMatching.jsp

```diff
@@ -38,20 +38,10 @@ should be included in file importing DeliveryMessages
   </t:dataList>
 
   <h:dataTable value="#{question.matchingArray}" var="matching">
-    <h:column rendered="#{delivery.feedback eq 'true' &&
-       delivery.feedbackComponent.showCorrectResponse && !delivery.noFeedback=='true'}">
-      <h:panelGroup id="image"
-        rendered="#{matching.isCorrect}"
-        styleClass="si si-check-lg" >
-      </h:panelGroup>
-      <h:panelGroup id="ximage"
-        rendered="#{!matching.isCorrect}"
-        styleClass="si si-remove feedBackCross" >
-      </h:panelGroup>
-      <h:graphicImage id="image2"
-        width="16" height="16"
-        alt="#{deliveryMessages.alt_incorrect}" url="/images/delivery/spacer.gif">
-      </h:graphicImage>
+    <h:column rendered="#{delivery.feedback eq 'true' && delivery.feedbackComponent.showCorrectResponse && !delivery.noFeedback=='true'}">
```

Comment:
I'm not sure if it's another issue but it _seems_ like this should also be ` && delivery.noFeedback ne 'true'` to be consistent with the delivery.feedback? Maybe this isn't a bug though but using eq and == doesn't seem correct.

Reply:
yes the mixing is puzzling! plus might as well be consistent and make delivery.feedbackComponent.showCorrectResponse eq 'true' also!